code/game/machinery/_machinery.dm

```diff
@@ -616,6 +616,8 @@ Class Procs:
 		inserted_scan_id.forceMove(drop_location())
 		if(!issilicon(user) && Adjacent(user))
 			user.put_in_hands(inserted_scan_id)
+		if(authenticated)
+			authenticated = 0
```

Comment:
LOGGED_OFF

Reply:
If I change it gives an error "LOGGED_OFF: undefined var". Do I have to define that in _machinery.dm?

Reply:
Oh right. You're gonna need to move those defines up the tree to the code/_DEFINES folder. It's easier to change the defines later if they're defined once rather than in a bunch of files.